internal/common/validation/podspec.go

```diff
@@ -8,7 +8,7 @@ import (
 	"k8s.io/component-helpers/scheduling/corev1/nodeaffinity"
 )
 
-func ValidatePodSpec(spec *v1.PodSpec, maxAllowedSize uint) error {
+func ValidatePodSpec(spec *v1.PodSpec, maxAllowedSize uint, minJobResources v1.ResourceList) error {
```

Comment:
Same comment as above. Should we pass in a larger object that contains all values needed to do the validation?